src/common/components/forms/OfferForm.tsx

```diff
@@ -75,13 +78,29 @@ const OfferForm: FC<OfferFormProps> = ({ nft, marketplace, refetch }) => {
 
     const offerAmount = Number(amount);
 
+    const newActions: Action[] = [
+      {
+        name: `Making offer of ${amount} SOL for ${nft.name}...`,
+        id: `makeOffer`,
+        action: onOffer,
+        param: offerAmount,
+      },
+    ];
+
     try {
-      await onOffer(offerAmount);
-      await refetch();
-      toast.success(`Confirmed offer success`);
-      return;
+      await runActions(newActions, {
+        onActionSuccess: async () => {
+          await refetch();
+          toast.success(`Confirmed offer success`);
+        },
+        onComplete: async () => {
+          await refetch();
+        },
+        onActionFailure: async () => {
+          await refetch();
+        },
+      });
     } catch (err: any) {
-      toast.error(err.message);
```

Comment:
This appears to never display the error. Intentional?

Reply:
an oversight thx for catching it